### PR TITLE
EFR32-related doc fixes

### PR DESCRIPTION
**Pull request by applecuckoo**
Mainly typos, I also added a silabs entry to the supported ports. Let me know if anything should be reworded. Thanks!

### CODE REVIEW

File: ports/silabs/README.md [hunk 1]
```diff
@@ -115,18 +114,18 @@ At the boot stage, two scripts will be run (if not booting in safe mode). First,
 
 After code.py has finished executing, a REPL prompt will be presented on the serial port. Other files can also be executed by using the **Thonny** editors or using **Ampy** tool.
 
-![Thony](./res/Thony.png)
+![Thonny](./res/Thonny.png)
 
 With the boards which support USB mass storage, we can drag the files to the board file system. However, because the EFR32 boards don’t support USB mass storage, we need to use a tool like **Ampy** to copy the file to the board. You can use the latest version of **Ampy** and its  command to copy the module directories to the board.
```
Comment (applecuckoo) [inline ports/silabs/README.md:119]: I saw the use of Ampy here, it's currently maintained by @scientifichackers but the attached learn guide seems outdated. Not sure if there's anything we can do here.

File: ports/silabs/README.md [hunk 2]
```diff
@@ -10,68 +11,69 @@ This port brings the Silicon Labs EFR32 series of MCUs to Circuitpython.
...
+Clean the project by using:
 
-    $ make BOARD=explorerkit_xg24_brd2703a clean
+    make BOARD=explorerkit_xg24_brd2703a clean
 
-## Bring-up on the board ##
+## Running CircuitPython ##
 
-### Getting a REPL prompt ###
+### Connecting to the Serial Console ###
```
Comment (applecuckoo) [inline ports/silabs/README.md:76]: I renamed this based on the Welcome to CircuitPython guide

**Review by applecuckoo — COMMENTED** [review]
Here are the things that I need feedback on:

File: README.rst [hunk 3]
```diff
@@ -237,6 +237,7 @@ nrf               stable
 raspberrypi       stable
 stm               ``F4`` stable | ``others`` beta
 unix              alpha
+silabs            alpha
```
Comment (microdev1) [inline README.rst:240]:
```suggestion
raspberrypi       stable
silabs            alpha
stm               ``F4`` stable | ``others`` beta
unix              alpha
```
Thanks @applecuckoo, a minor suggestion, alphabetize the ports.

File: ports/silabs/README.md (hunk 1 above)
Comment (tannewt) [reply ports/silabs/README.md:119]: Generally we suggest Thonny now.

**Comment by applecuckoo** [comment]
Okay, messed this up by not working on a branch. Will open a brand new PR once I sort this out...

**Comment by applecuckoo** [comment]
Succeeded by #8042

**Pull request closed without merging**